PeerioTechnologies/peerio-mobile
# fix: show shared with tiny avatars [branch ch8628]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/8628/mobile-shared-with-tiny-avatars-don-t-show
#### Testing instructions  
The tiny avatars show users youve already shared the folder with. Hence they can be found when you share a folder then try to share it again, the tiny avatars are at the bottom of the contact select view.

2 cases:
case 1: Share a folder with 4 users and check that tiny avatars are rendered correctly
case 2: Remove one user from that shared folder and check that tiny avatars are rendered correctly

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
